packages/gatsby/src/cache-dir/loader.js

```diff
@@ -299,6 +306,23 @@ const queue = {
         done()
       })
 
+      page.layoutComponentChunkName && getResourceModule(page.layoutComponentChunkName, (err, l) => {
+        if (err) {
+          console.log(`Loading the Layout for ${page.path} failed`)
```

Comment:
This pattern is copied the other resource fetching functions. I'm not sure what the correct approach is, but we should be consistent.